src/components/NewLocationPage/CountyMap/CountyMap.tsx

```diff
@@ -61,6 +61,24 @@ const MAP_TYPE_INFO: { [key in MapType]: MapTypeInfo } = {
   },
 };
 
+function useMapTileOffset(): number {
+  const [yPosition, setYposition] = useState(0);
```

Comment:
`yPosition` of what? I can't fully guess what this is without reading all of the code.  A fully verbose name might be `windowScrollYPosition`, but you could shorten to`windowScrollY` or `scrollY` and still be pretty clear.